wlauto/common/android/workload.py

```diff
@@ -667,6 +667,8 @@ def validate(self):
         super(AndroidUxPerfWorkload, self).validate()
         self.uiauto_params['package'] = self.package
         self.uiauto_params['markers_enabled'] = self.markers_enabled
+        if not self.activity:
+            self.activity = 'None'
```

Comment:
This will break the check in the rest of the python code as `if self.activity' will then always return true.